integration-tests/src/lib.rs

```diff
@@ -1,3 +1,10 @@
+#![cfg_attr(dylint_lib = "amplifier_lints", allow(ensure_msg_has_permissions))]
+#![cfg_attr(
+    dylint_lib = "amplifier_lints",
+    allow(restrict_cosmwasm_addr_in_msg_struct)
```

Comment:
```suggestion
    allow(cosmwasm_addr_in_msg_struct)
```
same reason as for the other lint